app/templates/app/project_detail.html

```diff
@@ -0,0 +1,82 @@
+{% extends "base.html" %}
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+<br/>
+<div class="project-detail">
+  <p><a href="{% url 'projects' %}">{% trans "Projects" %}</a> > {{ project.name }}</p>
+
+  <div class=project-div-row>
+
+    <div class="col-sm-6 project-detail-left-col">
+      <h1>{{ project.name }}</h1>
+      <a href="{{ project.url }}">{{ project.url }}</a>
+      <p>
+          <a href="{% url 'institution_detail' project.institution.id %}">
```

Comment:
I believe this causes an n+1 queries. Please check with assertNumQueries in a test for this page.

Reply:
will fix